common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1647,6 +1653,12 @@
     ],
     "sqlState" : "22009"
   },
+  "ILLEGAL_INTERVAL_ARGUMENT_VALUE" : {
```

Comment:
BTW, there is the `INVALID_PARAMETER_VALUE` error condition. I think yours new conditions should be in its sub-classes. Please, double check.

Reply:
I guess it could go. But I would need to change the error message as well. As the point here is we created interval and it is a valid interval, but just add interval expression can not use microseconds. So it might be the best to just change it INVALID_PARAMETER_VALUE.INTERVAL_WITH_MICROSECONDS. Do you agree on the naming?